test/integration/user_can_remove_items_from_cart_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.